### DISCUSSION TITLE
Demo Request

### DISCUSSION THREAD

**Discussion by nraxana** [cloudscape-design/components#2322] [Q&A] [answered]
We want to know more on the deployment and usage of Cloudscape tool. Could you please arrange a demo for this.?

Kindly reachout to noorulraxana@gmail.com for scheduling demo call.

**Comment by YueyingLu** [comment, marked as answer]
Hi @nraxana, 
thanks for reaching out.
We currently do not provide demo calls, here is the list of channels we offer to connect with Cloudscape https://cloudscape.design/about/connect/. Can we know more about your interest in Cloudscape?

Best regards,
Yueying

**Reply by nraxana** [reply]
Hi YueyingLu,

Could you please provide us the documentaion link on the deployment of Cloudscape to install?

Regards
Raxana

**Reply by gethinwebster** [reply]
https://cloudscape.design/get-started/integration/using-cloudscape-components/

**Reply by nraxana** [reply]
Hi @gethinwebster . Thank you for the documentation link.
Could you please let us know the VM specifications and requirements to install cloudscape?

**Reply by gethinwebster** [reply]
I'm not quite sure what you mean by VM specifications. Cloudscape is a frontend library. Anywhere that you can run [npm](https://www.npmjs.com/) you should be able to install Cloudscape.
